### DISCUSSION TITLE
AC3/E-AC3/DTS passthrough

### DISCUSSION THREAD

**Discussion by mdabbs** [google/oboe#1168] [Q&A] [answered]
Would Oboe support some type of bitstream passthrough if I use it on an Android TV device?

**Comment by philburk** [comment, marked as answer]
Oboe currently does not support bitstream passthrough for compressed audio.
But we are considering that for a future version. We may provide it as IEC61937 passthrough.

Currently, the PCM audio is scaled for volume control, and may undergo format conversion. So any bitstream would be corrupted.
